### PR TITLE
Updates on plots

### PR DESCRIPTION
**Pull request by davidwalter2**
- Fix event numbers in log files
- Support to specify legend padding location
- Various refinements of plots for paper submission

### CODE REVIEW

**Comment by kdlong** [comment]
Looks like the error is in my code. 

Probably it's coming from cases where there is no ratio? This might fix it

```python3
y_label_pos = min(*[get_ylabel_position(ax) for ax in filter(None, [ax1, *ratio_axes]]))
```

File: scripts/plotting/postfitPlots.py [hunk 1]
```diff
@@ -807,7 +832,10 @@ def make_plots(
     procs = [
         k.replace("expproc_", "").replace(f"_{fittype};1", "")
         for k in fitresult.keys()
-        if fittype in k and k.startswith("expproc_") and "hybrid" not in k
+        if fittype in k
+        and k.startswith("expproc_")
+        and "hybrid" not in k
```
Comment (cippy) [inline scripts/plotting/postfitPlots.py:837]: what if "and all(x not in k for x in ["hybrid", "masked"]) " to be prepared for additional conditions, and also to save one line

File: scripts/plotting/postfitPlots.py [hunk 2]
```diff
@@ -93,6 +93,12 @@
     default=None,
     help="Location in (x,y) for additional text, aligned to upper left",
 )
+parser.add_argument(
```
[the remark this answers is not in the document]
Comment (davidwalter2) [reply scripts/plotting/postfitPlots.py:96]: I think what matters most is the relative fraction between height and widths, and for this reason I think it is enough to set one of both.

**Comment by davidwalter2** [comment]
OK this PR would be good to be merged from my side. PR validation looks good:
this PR: https://cmsmwbot.web.cern.ch/WMassAnalysis/PRValidation/PR560/2024_11_11/
reference: https://cmsmwbot.web.cern.ch/WMassAnalysis/PRValidation/ReferenceRuns/2024_11_11_189aa48/

**Pull request merged**